rskj-core/src/test/java/co/rsk/peg/BridgeSupportReleaseBtcTest.java

```diff
@@ -152,52 +139,52 @@ void eventLogger_logReleaseBtcRequested_after_rskip_146_185() throws IOException
         verify(repository, never()).transfer(any(), any(), any());
         verify(eventLogger, times(1)).logReleaseBtcRequested(any(byte[].class), any(BtcTransaction.class), any(Coin.class));
         verify(eventLogger, times(1)).logReleaseBtcRequestReceived(any(), any(), any());
-        verify(eventLogger, times(0)).logReleaseBtcRequestRejected(any(), any(), any());
+        verify(eventLogger, never()).logReleaseBtcRequestRejected(any(), any(), any());
     }
 
     @Test
-    void eventLogger_logReleaseBtcRequested_after_rskip_146_185_326() throws IOException {
-        when(activationMock.isActive(ConsensusRule.RSKIP146)).thenReturn(true);
-        when(activationMock.isActive(ConsensusRule.RSKIP185)).thenReturn(true);
-        when(activationMock.isActive(ConsensusRule.RSKIP326)).thenReturn(true);
+    void eventLogger_logReleaseBtcRequested_release_before_papyrus_and_updateCollections_after_papyrus() throws IOException {
+        ActivationConfig.ForBlock wasabiActivations = ActivationConfigsForTest.wasabi100().forBlock(0L);
+        ActivationConfig.ForBlock papyrusActivations = ActivationConfigsForTest.papyrus200().forBlock(0L);
 
+        bridgeSupport = initBridgeSupport(eventLogger, wasabiActivations);
         bridgeSupport.releaseBtc(releaseTx);
 
+        bridgeSupport = initBridgeSupport(eventLogger, papyrusActivations);
         Transaction rskTx = buildUpdateTx();
         bridgeSupport.updateCollections(rskTx);
 
         verify(repository, never()).transfer(any(), any(), any());
-        verify(eventLogger, times(1)).logReleaseBtcRequested(any(byte[].class), any(BtcTransaction.class), any(Coin.class));
-        verify(eventLogger, times(1)).logReleaseBtcRequestReceived(any(), any(), any());
-        verify(eventLogger, times(0)).logReleaseBtcRequestRejected(any(), any(), any());
+        verify(eventLogger, never()).logReleaseBtcRequested(any(byte[].class), any(BtcTransaction.class), any(Coin.class));
+        verify(eventLogger, never()).logReleaseBtcRequestReceived(any(), any(), any());
+        verify(eventLogger, never()).logReleaseBtcRequestRejected(any(), any(), any());
     }
 
     @Test
-    void eventLogger_logReleaseBtcRequested_release_before_activation_and_updateCollections_after_activation() throws IOException {
-        when(activationMock.isActive(ConsensusRule.RSKIP146)).thenReturn(false);
-        when(activationMock.isActive(ConsensusRule.RSKIP185)).thenReturn(false);
+    void handmade_release_before_rskip_146_185() throws IOException {
```

Comment:
Let's use the naming test method we agreed on. Wdyt? The expected output or behavior is not clear to me.

Reply:
Added the expected behavior to the test name